instrumentation/rocketmq/rocketmq-client/rocketmq-client-5.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/rocketmqclient/v5_0/ReceiveSpanFinishingCallback.java

```diff
@@ -49,8 +49,14 @@ public void onSuccess(ReceiveMessageResult receiveMessageResult) {
               null,
               timer.startTime(),
               timer.now());
+      // For batch messages, each message should have its own context that properly
```

Comment:
As pointed out in https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/15523#discussion_r2587984217 this won't work.